packages/apps-config/src/api/spec/zero.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright 2017-2020 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+// structs need to be in order
+/* eslint-disable sort-keys */
+
+export default {
+  Address: 'AccountId',
```

Comment:
You would want `LookuoSource: 'AccountId'` as well (otherwise transfers won't work)

Reply:
done